extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -579,7 +586,7 @@ public String getErrorMessage() {
         try {
             return getTextFromObject(initialAjaxJson.getObject(2).getObject("playerResponse").getObject("playabilityStatus")
                     .getObject("errorScreen").getObject("playerErrorMessageRenderer").getObject("reason"));
-        } catch (ParsingException e) {
+        } catch (Exception e) {
```

Comment:
Why was this changed?

Reply:
So that it also catches NullPointerExceptions, which are generated when there is no error message. In that case `null` should be returned

Reply:
Then why aren't you catching `(ParsingException | NullPointerException e)`?